dune-project

```diff
@@ -1,8 +1,6 @@
-(lang dune 3.1)
+(lang dune 3.7)
```

Comment:
We're using features that will only be released with dune 3.8 🤔 I think we should make that explicit here.

Reply:
https://github.com/melange-re/melange-opam-template/pull/7/commits/092225f03781fb521394529901e2df9b8bd41a59